sources/packages/backend/apps/api/src/route-controllers/institution/_tests_/e2e/institution.aest.controller.getInstitutionNameOptions.e2e-spec.ts

```diff
@@ -0,0 +1,47 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import {
+  AESTGroups,
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getAESTToken,
+} from "../../../../testHelpers";
+import { E2EDataSources, createE2EDataSources } from "@sims/test-utils";
+
+describe("InstitutionAESTController(e2e)-getInstitutionNameOptions", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+  });
+
+  it("Should return an array of institutions when there are in institutions in the database.", async () => {
```

Comment:
Minor but the "in" seems to be lost in the sentence. Was it on purpose?
Can it be rephrased as "Should return an array of institutions when there are institutions in the database.".